packages/devtools/lib/src/service.dart

```diff
@@ -6,13 +6,51 @@ import 'dart:async';
 import 'dart:html' hide Event;
 import 'dart:typed_data';
 
+import 'package:sse/client/sse_client.dart';
+import 'package:vm_service/utils.dart';
+
 import 'vm_service_wrapper.dart';
 
-Future<VmServiceWrapper> connect(Uri uri, Completer<Null> finishedCompleter) {
-  final WebSocket ws = WebSocket(uri.toString());
+void _connectWithSse(
+  Uri uri,
+  Completer<VmServiceWrapper> connectedCompleter,
+  Completer<Null> finishedCompleter,
+) {
+  uri = uri.scheme == 'sse'
+      ? uri.replace(scheme: 'http')
+      : uri.replace(scheme: 'https');
+  final SseClient client = SseClient('$uri');
+  client.onOpen.listen((_) {
+    final Stream<String> stream = client.stream.asBroadcastStream();
+    stream.listen((_) {}, onError: (error) {
```

Comment:
How about `stream.drain().catchError` instead of `listen` with an empty callback

Reply:
Nice!